src/controllers/BaseController.sol

```diff
@@ -83,4 +94,11 @@ abstract contract BaseController is Ownable, Oval {
     function maxTraversal() public view override returns (uint256) {
         return maxTraversal_;
     }
+
+    /**
+     * @notice Max age of a historical price that can be used instead of the current price.
+     */
+    function maxAge() public view override returns (uint256) {
+        return maxTraversal_;
```

Comment:
a critical error here! should be `maxAge_`. found while fixing tests for CI

Reply:
Amazing!